src/Compilers/Core/Portable/Diagnostic/LocalizableString.cs

```diff
@@ -32,22 +34,22 @@ public string ToString(IFormatProvider formatProvider)
             }
         }
 
-        public static explicit operator string(LocalizableString localizableResource)
+        public static explicit operator string?(LocalizableString localizableResource)
         {
             return localizableResource.ToString(null);
```

Comment:
>localizableResource.ToString(null); [](start = 19, length = 35)

Does this ever return `null`?

Reply:
Now that I moved `GetText` to `string` i think not. 

---
In reply to: [356889553](https://github.com/dotnet/roslyn/pull/40268#discussion_r356889553) [](ancestors = 356889553)